{{cookiecutter.project_slug}}/clients/web/react/src/services/user/api.ts

```diff
@@ -9,61 +9,51 @@ import {
   userShapeWithToken,
 } from './models'
 
-const login = createCustomServiceCall(
-  {
-    inputShape: loginShape,
-    outputShape: userShapeWithToken,
-  },
-  async ({ client, input, utils }) => {
+const login = createCustomServiceCall({
+  inputShape: loginShape,
+  outputShape: userShapeWithToken,
+  cb: async ({ client, input, utils }) => {
     const res = await client.post('/login/', utils.toApi(input))
     return utils.fromApi(res.data)
   },
-)
+})
 
-const signup = createCustomServiceCall(
-  {
-    inputShape: userCreateShape,
-    outputShape: userShapeWithToken,
-  },
-  async ({ client, input, utils }) => {
+const signup = createCustomServiceCall({
+  inputShape: userCreateShape,
+  outputShape: userShapeWithToken,
+  cb: async ({ client, input, utils }) => {
     const res = await client.post('/users/', utils.toApi(input))
     return utils.fromApi(res.data)
   },
-)
+})
 
 const logout = createCustomServiceCall(async ({ client }) => {
   return client.post(`/logout/`)
 })
 
-const requestPasswordReset = createCustomServiceCall(
-  {
-    inputShape: forgotPasswordShape,
-  },
-  async ({ client, input }) => {
+const requestPasswordReset = createCustomServiceCall({
+  inputShape: forgotPasswordShape,
+  cb: async ({ client, input }) => {
     await client.post(`/password/reset/`, input)
   },
-)
+})
 
-const resetPassword = createCustomServiceCall(
-  {
-    inputShape: { userId: z.string(), token: z.string(), password: z.string() },
-    outputShape: userShape,
-  },
-  async ({ client, input, utils }) => {
+const resetPassword = createCustomServiceCall({
+  inputShape: { userId: z.string(), token: z.string(), password: z.string() },
+  outputShape: userShape,
+  cb: async ({ client, input, utils }) => {
     const { token, user_id, ...rest } = utils.toApi(input)
     const res = await client.post(`/password/reset/confirm/${user_id}/${token}/`, rest)
     return utils.fromApi(res.data)
   },
-)
+})
 
-export const userApi = createApi(
-  {
-    client: axiosInstance,
-    baseUri: '/users/',
-    models: {
-      create: userCreateShape,
-      entity: userShape,
-    },
-  },
-  { login, logout, requestPasswordReset, resetPassword, signup },
-)
+export const userApi = createApi({
+  client: axiosInstance,
+  baseUri: '/users/',
+  models: {
+    create: userCreateShape,
+    entity: userShape,
+  },
+  customCalls: { login, logout, requestPasswordReset, resetPassword, signup },
+})
```

Comment:
This all looks fine to me, but note that the bootstrapper is only capable to running and testing one app at a time. It detected more changes on the vue side so that is the version it deployed and ran tests against.

Ideally we would do separate PRs for each (vue, react) but I think this is fine as long as you've tested locally that the react version didn't introduce new linting or test errors. (doesn't look like it would)